sycl/test/regression/group.cpp

```diff
@@ -100,7 +100,7 @@ bool group__get_linear_id() {
   const range<DIMS> GlobalRange = LocalRange * GroupRange;
   using DataType = size_t;
   const int DataLen = GlobalRange.size() * DIMS;
-  std::unique_ptr<DataType> Data(new DataType[DataLen]);
+  std::unique_ptr<DataType[]> Data(new DataType[DataLen]);
```

Comment:
Good catch.
But on the other hand, just using `std::vector<DataType> Data(DataLen)` at the first place would have avoided this bug and avoided the next line.